app/routes/models/calculation.js

```diff
@@ -9,27 +8,37 @@ function GetBandText (band) {
 function toRow (results, property, formatType) {
   const data = []
   data.push({ text: GetBandText(results.band) })
-  results.result.map(x => data.push(
-    {
-      text: (formatType === 'currency'
-        ? `£${x[property].toFixed(2)}`
-        : `${Math.round(x[property] * 100)}%`),
-      format: 'numeric'
-    }))
-
-  return [...data, ...fillGaps(results, formatType)]
-}
-
-function fillGaps (results, formatType) {
-  const data = []
-  for (let i = 0; i < schemeYears.length - results.result.length; i++) {
+  results.result.map((x) => {
     data.push(
       {
         text: (formatType === 'currency'
-          ? '£0.00'
-          : '0%'),
+          ? `£${x[property].toFixed(2)}`
+          : `${Math.round(x[property] * 100)}%`),
```

Comment:
does this rounding undo any of the rounding we've got elsewhere in the application?

Reply:
I've removed the rounding, as it is now redundant. It was put in place before rounding had been implemented.

Reply:
The rounding is needed though on the percentage calculation.

`${Math.round(x[property] * 100)}%`

Example 150001 in 2024

amounts above £30,000.00 and no more than £50,000.00 = 55.00000000000001%